frontends/web/src/routes/account/info/info.jsx

```diff
@@ -104,8 +81,7 @@ export default class Info extends Component {
                         <div class="content padded flex flex-column flex-center">
                             <div class={style.infoContent}>
                                 <SigningConfiguration
-                                    t={t}
-                                    signingConfiguration={info.signingConfiguration} />
+                                    info={info} code={this.props.code} />
```

Comment:
this.props.code -> code

Reply:
why did you decide to pass `info` instead of `info.signingConfiguration` as before? as far as I can tell the component doesn't need anything else but `signingConfiguration`.

Reply:
Because we have an AccountInfo interface defined and now that the info is passed to a new tsx component, it can be typed and the SigningConfiguration component can check that the correct account info is being passed to it. 
I guess other option would be to define a new SigningConfiguration Interface that the AccountInfo would extend

Reply:
Because we have an AccountInfo interface defined and now that the info is passed to a new tsx component, it can be typed and the SigningConfiguration component can check that the correct account info is being passed to it.
I guess other option would be to define a new SigningConfiguration Interface that the AccountInfo would extend

Reply:
yep, that would be right.

`export interface SigningConfiguration` in signingconfiguration.tsx, which account.tsx  can import and use inside of AccountInfo.

Reply:
fixed